azure-pipelines.yml

```
@@ -185,7 +185,8 @@ stages:
             curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
             python3.8 get-pip.py && \
             pip3 --version && \
-            pip3 install setuptools wheel numpy==1.17.3 cython==0.29.18 pybind11 pytest pytest-timeout pytest-xdist pytest-env pytest-cov Pillow mpmath matplotlib==3.1.3 pythran --user && \
+            pip3 install setuptools==58.4.0 wheel --user && \
```

Any reason this is different from the other?
```suggestion
            pip3 install setuptools<58.5 wheel --user && \
```

---

I think this could fix failures in azure-pipelines.   Lets see.

---

BTW, "pip3 install setuptools<58.5 wheel --user &&" should do same (I'll use this in the final version).

The tricky part is why the pip deps resolver substitute setuptools>=58.5 (probably due to indirect deps in the matplotlib) even if we have strict requirement for setuptools==58.4.0.  That's why we can't install all things in one shot.  Seems to be a pip bug...